aten/src/ATen/cpu/vec256/missing_vst1_neon.h

```diff
@@ -0,0 +1,9 @@
+/* Workaround for missing vst1q_f32_x2 in gcc-8.  */
```

Comment:
Thanks Marcin for fixing this.
I see that you are working at RedHat.
Could you please submit a back-port of the following two patches to gcc-8 branch and maybe to gcc-9 branch as well?
```
git clone -b origin/releases/gcc-8 https://github.com/gcc-mirror/gcc.git
cd gcc 
git cherry-pick 568421baa5a4cdb7bb7c5ac323c939492ee3f052
git cherry-pick 391625888d4d97f9016ab9ac04acc55d81f0c26f
make -j64
make -j64 check
```
On my side the backport to gcc-7 was passing bootstrap and regression testing without any other changes.

gcc-7 branch is not maintained on the FSF branches, and I took care to propagate the changes to linux distros relying on that.
gcc-8 will release one more time by end of the year.  I cannot send the backports as I don't have the copyright assignment on file yet: still working to get that signed in my org.
Thanks!

Reply:
Thanks for hints.

Mailed RHEL mailing list. Will look at it more next week.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1876834 reported